applications/plugins/Compliant/Compliant_test/Assembly_test.cpp

```diff
@@ -414,8 +409,8 @@ struct Assembly_test : public CompliantSolver_test
         // The solver
         complianceSolver = addNew<OdeSolver>(solverObject);
         complianceSolver->storeDynamicsSolution(true);
-//        complianceSolver->f_printLog.setValue(true);
-//        complianceSolver->debug.setValue(true);
+        //        complianceSolver->f_printLog.setValue(true);
```

Comment:
I would say either removing the commented lines or removing this indent changes.

Reply:
Auto-align fault :)